src/components/atoms/Input/Input.component.tsx

```diff
@@ -1,25 +1,94 @@
-import React from 'react'
-import { Input as AntInput, InputRef as AntInputRef, InputProps as AntInputProps } from 'antd'
+import React, { ChangeEvent, useState } from 'react'
+import { Input as AntInput, InputProps as AntInputProps } from 'antd'
+
+import { CSSProperties } from 'styled-components'
 
 import { PreviewProps } from '@/types/preview'
 
 import { ComponentProps } from '@/types/component'
 
+import { getClasses, isStyleObject } from '../../../utils/common'
+
 import { Form, FormItemProps } from '../Form/Form.component'
 
-export interface InputProps extends PreviewProps, ComponentProps<AntInputProps> {
+import { Text } from '../Text/Text.component'
+
+import styles from './Input.module.css'
+
+export interface InputProps extends PreviewProps, ComponentProps<Omit<AntInputProps, 'style'>> {
   formItem?: boolean
   formItemProps?: Omit<FormItemProps, 'children'>
+  style?: Record<string, unknown> | string
```

Comment:
why don't we use `CSSProperties` here? 🤔

Reply:
yes it should be like that. but in Frontend repo, the data type of stype is NodeStyle, so I have to use Record here
<img width="552" alt="Screen Shot 2022-04-19 at 00 07 53" src="https://user-images.githubusercontent.com/99177620/163845032-41fc937c-2206-4899-8c1b-1b4074f43167.png">

Reply:
ah, we should consider refactoring it 😅

Reply:
yea =))